static/app/views/dashboardsV2/data.tsx

```diff
@@ -214,9 +219,10 @@ export const DASHBOARDS_TEMPLATES: DashboardDetails[] = [
   },
   {
     id: 'frontend-template',
-    title: t('Frontend'),
+    title: t('Frontend KPIs'),
```

Comment:
Ohhh should we try to avoid acronyms wherever possible? There is a non-zero amount of people that don't know what KPI stand for (I sometimes forget myself and have to look it up) so I'd vote for just removing it or replacing it with a different word